classes/02class/exercises/c02-network04/alanlima/scripts.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
```

Comment:
Nice script, but you'll see how this won't make a lot of sense during class 04..:)

Reply:
Thanks, I can see that. Creating bash scripts is quite good to learn the commands and for quick tests. But I can see a lot of problems in real case scenarios. 
I am looking forward to learning more about terraform ;)